backend/src/main/java/eventlistener/controller/NotificationUserController.java

```diff
@@ -31,4 +37,9 @@ public NotificationUser getSingleUser(@PathVariable String id){
         return notificationUserService.getSingleUser(id);
     }
 
+    @GetMapping("/event/{userId}")
+    public List<Event> getAllEventsFromUser(@PathVariable String userId){
```

Comment:
similiar to above, I think the proper rest way would be to do something like user/events